website/database/migrations/1740684140937_create_events_table.ts

```diff
@@ -19,6 +19,7 @@ export default class extends BaseSchema {
       table.integer('tickets_total').defaultTo(0)
       table.integer('tickets_remaining').defaultTo(0)
       table.decimal('price', 10, 2).defaultTo(0)
+      table.integer('product_id').unsigned().references('id').inTable('product_groups')
```

Comment:
This should not be done!!

Migrations are meant to be one-shot everytime they are "upped". Further changes should be included in a new migration. The simple argument is that, usually, migration runs are cached by name, so changing a migration itself and "upping" would not implement the change made. This might not be an issue in this specific case but keep this in mind.

Reply:
This can all be ignored by rolling back to just before this migration, but as a rule of thumb these files, once used for real, should not be modified.